unified-installer/astra-unified-installer.sh

```diff
@@ -87,10 +93,15 @@ readonly __PRODUCTION_AUTOSUPPORT_URL="https://support.netapp.com/put/AsupPut"
 readonly __GENERATED_CRS_DIR="./astra-generated"
 readonly __GENERATED_CRS_FILE="$__GENERATED_CRS_DIR/crs.yaml"
 readonly __GENERATED_OPERATORS_DIR="$__GENERATED_CRS_DIR/operators"
+readonly __GENERATED_OPERATORS_TRIDENT_DIR="$__GENERATED_CRS_DIR/operators/trident"
+readonly __GENERATED_OPERATORS_CONNECTOR_DIR="$__GENERATED_CRS_DIR/operators/connector"
 readonly __GENERATED_KUSTOMIZATION_FILE="$__GENERATED_OPERATORS_DIR/kustomization.yaml"
+readonly __GENERATED_TRIDENT_KUSTOMIZATION_FILE="$__GENERATED_OPERATORS_TRIDENT_DIR/kustomization.yaml"
+readonly __GENERATED_CONNECTOR_KUSTOMIZATION_FILE="$__GENERATED_OPERATORS_CONNECTOR_DIR/kustomization.yaml"
+
 readonly __GENERATED_PATCHES_TORC_FILE="$__GENERATED_CRS_DIR/post-deploy-patches_torc"
 readonly __GENERATED_PATCHES_TRIDENT_OPERATOR_FILE="$__GENERATED_OPERATORS_DIR/post-deploy-patches_trident-operator"
-readonly __GENERATED_TRIDENT_ACP_SECRET_FILE="$__GENERATED_OPERATORS_DIR/trident-acp-secret.yaml"
+readonly __GENERATED_TRIDENT_ACP_SECRET_FILE="$__GENERATED_CRS_DIR/trident-acp-secret.yaml"
```

Comment:
The main additions here are the specific `TRIDENT` and `CONNECTOR` directories and kustomization files. The split was necessary because namespace transformers don't have the ability to differentiate connector/trident resources. This means we have:

- `operators/kustomization.yaml` - includes the two next files as resources
    - `operators/connector/kustomization.yaml` - pulls in the astra-connector-operator kustomization file up in github
    - `operators/trident/kustomization.yaml` - pulls in the trident kustomization file up in github